hnsecure/src/main/resources/application.properties

```diff
@@ -16,6 +16,7 @@ issuer = https://common-logon-dev.hlth.gov.bc.ca/auth/realms/v2_pos
 valid-receiving-facility = BC00002041,BC00002047,BC00001013
 processing-domain = D
 version = 2.1
+is-filedrops-allowed = Y
```

Comment:
I'm wondering if is-filedrops-enabled might be more semantically accurate? Also, I prefer true/false for boolean properties as it better reflects how it's used in the code. When you load the property you can just convert using Boolean.valueOf().

Reply:
Did changes. comparing with Boolean.True